tests/appsec/test_automated_login_events.py

```diff
@@ -7,7 +7,7 @@
 
 @rfc("https://docs.google.com/document/d/1-trUpphvyZY7k5ldjhW-MgqWl0xOm7AMEQDJEAZ63_Q/edit#heading=h.8d3o7vtyu1y1")
 @coverage.good
-@released(cpp="?", golang="?", java="?", nodejs="4.4.0", dotnet="2.32.0", php="0.89.0", python="?", ruby="?")
```

Comment:
we should keep the original release, and add a `@bug(from 4.4.0 to 4.9.0)` maybe ?